app/public/html/setup.twig

```diff
@@ -0,0 +1,24 @@
+{% extends 'base.twig' %}
+
+{% block title %}Enter account data{% endblock %}
+
+{% block content %}
+    <h1>Choose data source</h1>
+    <form action="." method="post">
+        <input type="hidden" name="step" value="{{ next_step }}">
+        <fieldset class="mt-3">
+            <legend>Data Source</legend>
+            <div class="form-group">
+                <label for="bank_sfa">Choose file from configurations folder or </label>
+                <select class="form-control" id="data_collect_mode" name="data_collect_mode" required>
+                    {% for file in files %}
+                        <option value="{{ file }}">{{ file }}</option>
+                    {% endfor %}
+                    <option value="createNewDataset">Configure Dataset</option>
+                </select>
+                <small class="form-text text-muted">Choose an configuration file or configure it manual.</small>
```

Comment:
I believe the wording is a little bit confusing: "Configure Dataset" sounds like one creates a configuration file, even tho no configuration file will be generated.
What do you think about the following:
Label: `Configuration`
createNewDataset-Option: `Manually fill out the configuration form`
Help: `Choose a method to configure the importer. You can either select a configuration file that you created previously, or you can use the configuration form.`

Reply:
Yeah, change the wording as you like, if we implement a option to save new configuration files later, we can still change it.